packages/studio-plugin/src/ParsingOrchestrator.ts

```diff
@@ -87,6 +89,18 @@ export default class ParsingOrchestrator {
     return newModuleFile;
   }
 
+  getLayoutFile(layoutName: string): LayoutFile {
+    const layoutFile = this.layoutNameToLayoutFile[layoutName];
+    if (layoutFile) {
+      return layoutFile;
+    }
+    return new LayoutFile(
+      upath.join(this.paths.layouts, layoutName + ".tsx"),
+      this.getFileMetadata,
```

Comment:
We pass getFileMetadata into the layout file constructor, but it doesn't look like we have a case in `getFileMetadata` to handle layouts.  Should we add that?

Reply:
We don't need a case for layouts in `getFileMetadata` because `LayoutMetadata` isn't a type of `FileMetadata`. A component tree will never have a "component" in it that corresponds to a layout. Instead, we'll just be copying the components (i.e. some kinds of `ComponentState`) from a layout's component tree into a page's component tree